buildpacks/php/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Fix `composer.lock` parsing when "dist" key contains a `"type": "path"`. ([#176](https://github.com/heroku/buildpacks-php/pull/176))
+- All raw file system errors now include the filenames via the `fs-err` crate. ([#174](https://github.com/heroku/buildpacks-php/pull/174))
```

Comment:
I think there is one more usage in `buildpacks/php/src/php_project.rs`:
https://github.com/search?q=repo%3Aheroku%2Fbuildpacks-php+%2F%5Cbfs%5Cb%2F&type=code

Reply:
Great catch, updated. I was searching for `std::fs` so missed the `std::{fs` the `/\b\fs\b/` is a good trick